zhaquirks/sinope/sensor.py

```diff
@@ -37,7 +39,9 @@ class SinopeManufacturerCluster(CustomCluster):
     name = "Sinopé Manufacturer specific"
     ep_attribute = "sinope_manufacturer_specific"
     attributes = {
+        0x0003: ("firmware_number", t.uint16_t, True),
         0x0004: ("firmware_version", t.CharacterString, True),
+        0x0200: ("unknown", t.bitmap32, True),
```

Comment:
This isn't ideal, but I guess it's fine for now.
Or maybe something like ``unknown_attr_1`` would be better?